package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "brand-pearson.com",
```

Comment:
Shouldn't this name follow a format similar to https://github.com/openedx/brand-openedx/blob/master/package.json#L2?

Reply:
That's related to the scope and the registry: https://docs.npmjs.com/cli/v6/using-npm/scope we are not adding support to that yet. @anfbermudezme